python/lsst/pipe/tasks/diff_matched_tract_catalog.py

```diff
@@ -333,23 +351,29 @@ def validate(self):
             raise ValueError("\n".join(errors))
 
 
+@deprecated(reason="This class is no longer being used and will be removed after v28.",
+            version="v28.0", category=FutureWarning)
 @dataclass(frozen=True)
 class MeasurementTypeInfo:
     doc: str
     name: str
 
 
+@deprecated(reason="This class is no longer being used and will be removed after v28.",
+            version="v28.0", category=FutureWarning)
 class MeasurementType(Enum):
-    DIFF = MeasurementTypeInfo(
+    DIFF = SimpleNamespace(
         doc="difference (measured - reference)",
         name="diff",
     )
-    CHI = MeasurementTypeInfo(
+    CHI = SimpleNamespace(
         doc="scaled difference (measured - reference)/error",
         name="chi",
     )
```

Comment:
I don't object to the switch to `SimpleNamespace`, in fact, I rather prefer it. However, as this class is being deprecated and ultimately removed, I'm curious why the change?

Reply:
Leaving it as-is led to deprecation warnings on module import. I couldn't make a minimum reproducible example to debug the problem and so I just gave up and fixed it the easiest way I could.